src/Parser/Mage/Fire/Modules/Items/SoulOfTheArchmage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class SoulOfTheArchmage extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasFinger(ITEMS.SOUL_OF_THE_ARCHMAGE.id);
+    console.log(this.combatants.selected.hasFinger(ITEMS.SOUL_OF_THE_ARCHMAGE.id));
```

Comment:
The log here is unnecessary (or prepend it with `debug &&`).

Reply:
Removed in 1c149be